lib/styles/Styles.dart

```diff
@@ -26,21 +26,23 @@ ButtonStyle mainPageButtonStyle = OutlinedButton.styleFrom(
     padding: const EdgeInsets.all(16),
     tapTargetSize: MaterialTapTargetSize.padded,
     backgroundColor: primaryRed,
-    shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(8))
-);
+    shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)));
 
 class MainPageButton extends StatelessWidget {
   final VoidCallback onPressed;
   final Widget child;
   final EdgeInsets margin;
   final Color backgroundColor;
+  final bool circular;
+  final Size size;
 
   const MainPageButton({
```

Comment:
I would personally prefer to have 2 different classes for 2 different buttons (The circle one could extend the other one) because it can become really messy if we start to add new and new parameters. I'm used to C# and I would not allow it there but I understand cleancode in flutter can differ.

Reply:
I think this is a good point. I'll change it